tools/jenkins-slave-creation-unix/conf-ubuntu-gpu/install.sh

```diff
@@ -89,9 +101,23 @@ curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.li
   sudo tee /etc/apt/sources.list.d/nvidia-docker.list
 sudo apt-get update
 
-# Install nvidia-docker2 and reload the Docker daemon configuration
-sudo apt-get install -y nvidia-docker2
-sudo pkill -SIGHUP dockerd
+# Install nvidia docker related packages and reload the Docker daemon configuration
+# Install nvidia-container toolkit and reload the Docker daemon configuration
+# Refer Nvidia Docker : https://github.com/NVIDIA/nvidia-docker
+sudo apt-get install -y nvidia-container-toolkit
+sudo systemctl restart docker
+
+# Install & add nvidia container runtime to the Docker daemon
+# Refer https://github.com/nvidia/nvidia-container-runtime#docker-engine-setup
+sudo apt-get install nvidia-container-runtime
+sudo mkdir -p /etc/systemd/system/docker.service.d
+sudo tee /etc/systemd/system/docker.service.d/override.conf <<EOF
+[Service]
+ExecStart=
```

Comment:
Duplicate line?

Reply:
https://github.com/nvidia/nvidia-container-runtime#docker-engine-setup

Reply:
They should fix it as well?

Reply:
Addressed here and https://github.com/NVIDIA/nvidia-container-runtime/pull/97
Thanks for pointing out.

Reply:
Actually Felix pointed it out that empty string assignment resets the previous assignments

https://www.freedesktop.org/software/systemd/man/systemd.service.html

>    If the empty string is assigned to this option, the list of commands to start is reset, prior assignments of this option will have no effect.